docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/listing/list_item_macros.twig

```diff
@@ -222,18 +222,17 @@
     {{- title -}}
   </a>
   {% if media_link_type == "Infographic" or media_link_type == "Video" %}
-    <span class="media-type">({{ media_link_type|trans }})</span>
+    <span class="usa-collection__mediatype">({{ media_link_type|trans }})</span>
   {% endif %}
   {% if media_link_type == "File" %}
-    <span class="filetype {{file_type}}">
+    <span class="usa-collection__filetype">
```

Comment:
It looks like you missed a comment https://github.com/NCIOCPL/cgov-digital-platform/pull/4903#discussion_r2325310192